packages/components/src/ErrorState/ErrorState.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
-import { Stack, StackItem } from '@patternfly/react-core';
 import { ExclamationCircleIcon } from '@patternfly/react-icons/';
 import DefaultErrorMessage from './DefaultErrorMessage';
-import propTypes from 'prop-types';
-import { Title, EmptyState, EmptyStateVariant, EmptyStateIcon, EmptyStateBody, Button } from '@patternfly/react-core';
+import {
+  Title,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Button,
+  EmptyStateProps,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
 import './error-state.scss';
 
-const ErrorState = ({ errorTitle, errorDescription, ...props }) => {
+const ErrorState = (errorTitle: string, errorDescription?: string, ...props: EmptyStateProps[]) => {
```

Comment:
Component props must be enclosed in curly brackets:
```TSX
const ErrorState = ({errorTitle: string, errorDescription?: string, ...props}) => {
```

Also can you please extrac the type definitions into an interface and export it? You can follow this as an example: https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/InsightsLabel/InsightsLabel.tsx#L24

It should extend the `EmptyStateProps` interface from PF